### DISCUSSION TITLE
Please update manifest.json

### DISCUSSION THREAD

**Discussion by konstantinkoeppe** [deployphp/deployer#3939] [Bugs] [answered]
### Deployer Version

v7.5.3

### Target OS

Ubuntu 22.04

### Which PHP version are you using?

PHP 8.3

### Steps to reproduce

Please update https://deployer.org/manifest.json to include all recently released 7.5.x versions.

[This popular GitHub action](https://github.com/shivammathur/setup-php) to set up PHP environments queries for versions using the GitHub releases API and then expects to find them in the manifest.json file. Unfortunately the recently released versions are not included there (yet), which leads to broken CI/CD pipelines.

**Comment by antonmedv** [comment]
But how not having 7.5.3 breaks ci? manifest has other versions too. Currently latest in manifest is 7.4.1.

**Reply by konstantinkoeppe** [reply]
I am not sure why it is done this way (I am not the author of the GitHub Action), but it seems to query for releases using the GitHub releases API and then just looks for the corresponding download URL in the manifest file. We stumbled upon the problem because suddenly a few of our pipelines kept failing and found this to be the reason.

**Reply by antonmedv** [reply]
I see. Looks like the action downloads from deployer.org/deployer.phar url. But this URL stopped working starting with latest's release.

**Comment by antonmedv** [comment, marked as answer]
Done!
